### ISSUE TITLE
ClaimKit issues with Plethora manipulators

### ISSUE THREAD

**Issue by MarcelDev** [SwitchCraftCC/issues#81]
### Describe the bug

Destroying a manipulator in a claim that you are not trusted in, removes all of the modules in it (the modules are still there on the server side)

Similar: #66 & possibly #10

(pretty much the bug is only visual)

### To reproduce

get someone to place a manipulator and insert a module in a claim you are not trusted in
destroy the manipulator
the modules in it will dissapear on the client side

### Expected behavior

The modules should not dissapear.

### Relevant log output

```sh
No relevant log outputs i can find.
```

### Please confirm

- [X] This is not an exploit

**Comment by MarcelDev** [comment]
also happens with 3d prints but they become invisible

the modules are visible on the client side after exiting the chunk and coming back

**Comment by hugeblank** [comment]
> also happens with 3d prints but they become invisible

Yeah this is just a general `BlockEntity` issue. Also I said the same thing in your post about the monitors:
https://github.com/SwitchCraftCC/issues/issues/66#issuecomment-1485437836
